test/e2e/artifact_test.go

```diff
@@ -196,4 +175,160 @@ var _ = Describe("Podman artifact", func() {
 		podmanTest.PodmanExitCleanly([]string{"artifact", "inspect", artifactDigest[:12]}...)
 
 	})
+
+	It("podman artifact simple add --append", func() {
+		artifact1File, err := createArtifactFile(1024)
+		Expect(err).ToNot(HaveOccurred())
+
+		artifact2File, err := createArtifactFile(2048)
+		Expect(err).ToNot(HaveOccurred())
+
+		artifact3File, err := createArtifactFile(4192)
+		Expect(err).ToNot(HaveOccurred())
+
+		layersNames := map[string]int{
+			filepath.Base(artifact1File): 0,
+			filepath.Base(artifact2File): 0,
+			filepath.Base(artifact3File): 0,
+		}
+
+		artifact1Name := "localhost/test/artifact1"
+		podmanTest.PodmanExitCleanly([]string{"artifact", "add", artifact1Name, artifact1File}...)
```

Comment:
```suggestion
		podmanTest.PodmanExitCleanly("artifact", "add", artifact1Name, artifact1File)
```
similarly throughout.